multitemporal/test/test_1.py

```diff
@@ -36,6 +37,32 @@ def test_passthrough(nproc, tmpdir):
     # both pytest & numpy have approximate-equality abilities if needed
     assert (expected == actual).all()
 
+
+test_passthrough2_args = {
+    'compthresh': 0.01, # so smaller dataset will work
+    "dperframe": 1,
+    "sources"  : [{"name": "ndvi", "regexp": "^(?P<datestr>\\d{7})_L.._ndvi-(?P<tileid>\w{3}).tif$", "bandnum": 1}],
+    "steps"    : [{"module": "passthrough", "params": [], "inputs": ["ndvi"], "output": True}]}
+
+
+@pytest.mark.parametrize("nproc", [1, 2])
+def test_passthrough2(nproc, tmpdir):
```

Comment:
Copypasta is bad.  Extend the parametrization scheme instead so the existing test covers more cases.

Also, if you do, then `test_passthrough2` is going away as a name, but know that it's a much better idea to name your tests verbosely so people can see what went wrong in test reports.  Long, sloppy names aren't so bad for tests, eg, `test_passthrough_this_time_with_weird_regex`.

Reply:
PS you can instead move some common code into a fixture and call it from both, if the two tests are different enough to warrant it.